### ISSUE TITLE
[Bug]: Shell check ignore is not working

### ISSUE THREAD

**Issue by lamyergeier** [koalaman/shellcheck#2934]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1088
- My shellcheck version (`shellcheck --version` or "online"): 0.9.0
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit - Doesn't shows the error!

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/false
# shellcheck shell=bash
#shellcheck disable=SC2275,SC1036,SC1088
=() {
	# Refer: [Any command line calculator for Ubuntu? - Ask Ubuntu](https://askubuntu.com/questions/378661/any-command-line-calculator-for-ubuntu)
	ScriptP="$(readlink -f "${BASH_SOURCE[0]}")"
	if _FrameworkFunction "${ScriptP}" "${@}"; then return 0; fi

	local IFS=' '
	local calc="$*"
	# Uncomment the below for (p → +) and (x → *)
	#calc="${calc//p/+}"
	#calc="${calc//x/*}"
	printf '%s\n quit' "$calc" | gcalccmd | sed 's:^> ::g'
}
alias Calculator='='

```

#### Here's what shellcheck currently says:

```
Parsing stopped here. Invalid use of parentheses?
```

#### Here's what I wanted or expected to see:

- ignore the error
